articles/azure/azs-how-deploy-sql-template-powershell.md

```diff
@@ -36,11 +36,11 @@ The following quick-start templates are available on [GitHub](https://aka.ms/Azu
 
 Prerequisites from a Windows-based external client are:
 
-- PowerShell 5.1 and AzureStack PowerShell Module
+- PowerShell 5.1 and AzureStack PowerShell Module:
 
   - [Configure PowerShell Environment and Azure Stack Module](azs-how-configure-powershell-users.md)
 
-- Active Azure *Subscription* (required to create SPN if you want to use the same SPN for both Azure and Azure Stack)
+- An active Azure Stack *Subscription* (required to create an [SPN](azs-how-create-spn-portal.md) on Azure Stack).
```

Comment:
The section in the brackets is slightly redundant; you need the subscription to create any of the ARM template resources as well.

Reply:
`An active Azure Stack *Subscription*. Required to create an [SPN](azs-how-create-spn-portal.md) on Azure Stack`

I have just removed the brackets. As SPN is still required.